AlphaWallet.xcodeproj/project.pbxproj

```diff
@@ -4813,7 +4813,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ \"${CONFIGURATION}\" = \"Debug\" ]; then\n    \"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" \"$SRCROOT\"\nfi\n";
+			shellScript = "if [ \"${CONFIGURATION}\" = \"Debug\" ]; then\n    \"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" \"$SRCROOT/AlphaWalletTests\"\nfi\n";
```

Comment:
And can you change ths to the script approach like we just discussed in the issue?

Reply:
You want me to list each test file or can I just include the whole directory?

Reply:
Each file; for now at least, I think we'll only switch to directories when we cover all the files in a directory. In this case since we are going to just select a few for the PR, definitely list the files.

Reply:
I've used the Xcode file lists facility. All the files that are to be SwiftFormatted are in the file `SwiftFormatFiles.xcfilelist`